components/eamxx/src/share/iop/intensive_observation_period.cpp

```diff
@@ -268,9 +268,9 @@ initialize_iop_file(const util::TimeStamp& run_t0,
   scorpio::read_var(iop_file,"lon",&iop_file_lon);
 
   const Real rel_lat_err = std::fabs(iop_file_lat - m_params.get<Real>("target_latitude"))/
-                             m_params.get<Real>("target_latitude");
+                             std::max(m_params.get<Real>("target_latitude"),0.1);
```

Comment:
We need to cast to `Real` because `std::max` requires both arguments to be the same type.
```suggestion
                             std::max(m_params.get<Real>("target_latitude"),(Real)0.1);
```